build/cli.version

```diff
@@ -1 +1 @@
-1.0.0
+2.0.0-preview1-005418
```

Comment:
Does this build fine on the 3 test platforms? We should fix the mirroring code to auto update this rather than managing this manually.